configure.ac

```diff
@@ -779,8 +779,12 @@ AC_FUNC_STRERROR_R
 
 SAVE_CFLAGS="$CFLAGS"
 CFLAGS="-Wall -Werror"
-SAVE_LDFAGS="$LDFLAGS"
+SAVE_LDFLAGS="$LDFLAGS"
 LDFLAGS=""
+if test "x$ac_system" = "xWindows"; then
+  # This is exported from build.sh
+  LDFLAGS="$LDFLAGS -L${GNULIB_DIR}"
```

Comment:
This will only be used for the strtok test. LDFLAGS is later reset to SAVE_LDFLAGS. Is that what you want?

Reply:
Yes, "-L${GNULIB_DIR}" is added to LDFLAGS in build.sh, so it's saved with the other flags in SAVE_LDFLAGS.